service/src/main/resources/application.yml

```diff
@@ -29,7 +28,7 @@ drshub:
       name: NCI Cancer Research / Proteomics Data Commons (CRDC / PDC)
       hostRegex: '.*\.datacommons\.io'
       metadataAuth: false
-      bondProvider: dcf_fence
+      ecmFenceProvider: dcf_fence
```

Comment:
I think fence providers need to be `kebab-case` vs `snake_case` to match ECM's configs

Reply:
DrsHub maps these to kebab-case values: https://github.com/DataBiosphere/terra-drs-hub/blob/1ef7f66ce9b92d28c86db7bf993e6402d08d5649/service/src/main/java/bio/terra/drshub/models/ECMFenceProviderEnum.java#L4. I can change this to be consistent though!

Reply:
Oh actually a dash is not allowed in the enum name. This is what it looks like in ECM:
```
public enum Provider {
    RAS("ras"),
    GITHUB("github"),
    FENCE("fence"),
    DCF_FENCE("dcf-fence"),
    KIDS_FIRST("kids-first"),
    ANVIL("anvil");
...
}
```

Reply:
ooooh, and that's whats sent to ECM!